spanner/spanner_snippets/spanner/pg_spanner_connect_to_database.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+// [START spanner_postgresql_create_clients]
+
+import (
+	"context"
+	"io"
+
+	"cloud.google.com/go/spanner"
+	database "cloud.google.com/go/spanner/admin/database/apiv1"
+)
+
+// pgCreateClients shows how to create a database and admin client for a Spanner
+// PostgreSQL database.
+func pgCreateClients(w io.Writer, db string) error {
```

Comment:
Is this example needed? Other examples implicitly show this.

Reply:
Good point. You can nest this region tag inside one of the other samples.